config/application.rb

```diff
@@ -55,6 +55,8 @@
 
 require 'discourse_fonts'
 
+require_relative '../lib/zeitwerk/zeitwerk.rb'
```

Comment:
minor but can this just be `../lib/zeitwerk`? Not sure if we need a folder for it.